### DISCUSSION TITLE
where is "Display position always in center" in Osmand 4.0.7

### DISCUSSION THREAD

**Discussion by AminSadighi** [osmandapp/OsmAnd#12900] [Maps Related]
Hi
in Osmand 3.4.6 , we can chang the position in the display, by this menu
main menu >> configuration screen (at bottom) >> = Display position always in center (on / off )
I couldn't find this menu "Display position always in center"  in 4.0.7 in android

**Comment by dmpr0** [comment]
Hi!
You can find it in Menu – Settings – Selected profile – General settings – Display position always in center.

**Reply by AminSadighi** [reply]
Thanks a lot
